frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -24,6 +24,9 @@ import { capitalizeFirstLetter, compactNumber } from 'lib/utils'
 import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
 import { ProductPricingModal } from './ProductPricingModal'
 import { PlanComparisonModal } from './PlanComparison'
+import posthog, { Survey } from 'posthog-js'
```

Comment:
I'm trying to import the `Survey` type from posthog-js because the one exported in `frontend/src/types.ts` doesn't quite match and is resulting in type errors in the `getActiveMatchingSurveys` callback below. What's the best way around this @liyiy ?

Reply:
Probably best to just create a new `APISurvey` type that mirrors the values in posthog-js, you should also be able to use `pick` or `omit` to easily do that in the `types.ts` file

Reply:
That isn't really ideal Li, I think this is a problem in our package, we're not exporting the Survey types. I'll fix this on Monday @xrdt

Reply:
Thanks. 🙏